curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657f01ae9aea647b27402d3e.md

```diff
@@ -7,7 +7,10 @@ dashedName: step-10
 
 # --description--
 
-Now all that is left to complete this function is to return the `clean_snake_cased_string` from the function. So, go ahead and return the string by adding a `return` statement on the same level as the `clean_snake_cased_string` variable.
+To wrap up the function, return the `clean_snake_cased_string`. This will complete the function and allow you to use it to convert strings from pascal or camel case to snake case.
+
+Add a `return` statement at the end of the function to return the `clean_snake_cased_string`.
```

Comment:
this project is missing interactivity... I think it would be better to start with an example function call so that we can print step by step what is happening inside the function, instead of arriving at step 10 with no output in the terminal

Reply:
Up to this point, the steps outlined have primarily focused on constructing the function itself.  Given that each step involves building upon the function without any substantial changes in output, I think printing the same output at each step would not provide significant value to the learner.

However, starting from step 11 onward, the project transitions into developing the `main()` function where learners can now see the function's behavior, providing a more interactive experience